docs/pages/guides/webhooks.mdx

```diff
@@ -54,18 +58,110 @@ which events make the most sense. The Event Catalog provides a list of events
 available for subscription, along with their schema.
 
 Events available for use include:
+
 - UserEntered/UserLeft
 - StorageUpdated
 - RoomCreated/RoomDeleted
 - MaxConnectionsReached
 
+#### UserEntered
+
+```ts
+// Schema
+type UserEntered = {
+	type: "userEntered"
+	data: {
+		appId: string
+		roomId: string
+	  connectionId: number
+		userId: string | null
+		userInfo: Record<string, any> | null
+		enteredAt: string
+		numActiveUsers: number
+	}
+}
+
+// Example
+const userEnteredMessage = {
+	type: "userEntered",
+	data: {
+		appId: "aasdbe234bsdf"
+		roomId: "my-room-1"
+		connectionId: 4
+		userId: '1234sdgs'
+		userInfo: null
+		enteredAt: '2021-10-06T01:45:56.558Z'
+		numActiveUsers: 8
+	}
+}
+```
+
+#### UserLeft
```

Comment:
Could we have a short description of what it is in human language?

Reply:
good catch, added